src/main/java/io/jenkins/plugins/mcp/server/extensions/BuildLogsExtension.java

```diff
@@ -125,17 +132,18 @@ private BuildLogResponse getLogLines(Run<?, ?> run, long skip, int limit) throws
             limit = Math.abs(limit);
         }
 
-        log.debug(
-                "call with skip {}, limit {} for linesNumber {} with read with skip{}, limit {}",
-                skipInit,
-                limitInit,
-                linesNumber,
-                skip,
-                limit);
-
+        start = System.currentTimeMillis();
```

Comment:
It's better to log a start and log an end than compute by the code.

Anyway, you decide.

Reply:
not easy to correlate when you have a lot of requests in parallel and not the thread ID per default in the message as it's not the case in Jenkins logs